src/styles/font-jrkf/iconfont.js

```
@@ -0,0 +1,68 @@
+          document.write(
+            '<style>.svgfont {display: inline-block;width: 1em;height: 1em;fill: currentColor;vertical-align: -0.1em;font-size:16px;}</style>'
+          )
+        } catch (a) {
+          console && console.log(a)
+        }
+      }
+      ;(h = function () {
+        var a,
+          l = document.createElement('div')
+        ;(l.innerHTML = o._iconfont_svg_string_3744858),
+          (l = l.getElementsByTagName('svg')[0]) &&
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)